functions/twist_angle.h

```diff
@@ -24,4 +24,23 @@ class TwistAngleSVF {
 template<int N = 2, int OFFSET = 0>
 using TwistAngle = SingleValueAdapter<TwistAngleSVF<N, OFFSET>>;
 
+// Usage: TwistAcceleration<MAX>
+// Returns 0-32768 based on acceleration of twist in one direction
+// MAX : Maximum acceleration needed to return 32768
+// returned value: FUNCTION, same for all leds
+
+template<class MAX>
+class TwistAccelerationSVF {
+ public:
+  void run(BladeBase* blade) { max_.run(blade); }
+  int calculate(BladeBase* blade) {
+   float v = fusor.twist_accel() * 100 / max_.calculate(blade);
```

Comment:
Why times 100?
I think twist_accel() is in degrees per second per second. (or is it in radians per second per second?)
If it is in degrees per second per second, maybe we should multiply by 32768 / 360, as that would make 32768 mean one rotation per second per second.

Reply:
gyro_slope().x was returning 0 ~ 32000 values when I added a print, but to allow users to lower the threshold I have it multiplied by 100 so we can divide by MAX for customization.  I can try 32768 / 360 and update.

Reply:
updated both files and tested and it's working as envisioned.